Dockerfile.pipelines

```diff
@@ -1,15 +1,8 @@
-FROM ubuntu:xenial
+FROM ubuntu:bionic
 
 # This DockerFile is looked after by
 MAINTAINER Tim Greaves <tim.greaves@imperial.ac.uk>
 
-# Add the ubuntu-toolchain-r test ppa
```

Comment:
is this a consequence of not using xenial?

Reply:
Yes - toolchain-r drops all except 9 when you move to xenial. But you do get 9 - is this useful to test against?

Reply:
Dropped this change - see thoughts below on gcc versioning.